src/forms/FormFieldAddress/AddressDrawer/AddressDrawer.tsx

```diff
@@ -288,7 +294,7 @@ const AddressDrawer = (props: AddressDrawerProps): ReactElement => {
 					label: "States",
 					size: "sm",
 					inputSettings: {
-						options: listOfStates,
+						getOptions: () => getOptionsStates(state.data.country?.value),

```

Comment:
Will ``state.data.country`` have always a value? If not we should use an optional chaining operator to avoid an error.

```
getOptions: () => getOptionsStates(state.data?.country?.value)
```

Reply:
Data will always exist, data.country might not, but it already has an optional chaining operator